ChangeLog

```diff
@@ -36,6 +36,7 @@ ver. 0.10.3-dev-1 (20??/??/??) - development edition
 
 ### Fixes
 * `filter.d/exim.conf`: failregex extended - SMTP call dropped: too many syntax or protocol errors (gh-2048);
+* `action.d/badips.py`: solve an issue which made ban to fail under FreeBSD (gh-2059);
```

Comment:
Good catch, but ...

something more precise, like:
```diff
-* `action.d/badips.py`: solve an issue which made ban to fail under FreeBSD (gh-2059);
+* `action.d/badips.py`: implicit convert IPAddr to str, solves an issue "expected string, IPAddr found" (gh-2059);
```
would be appreciated here.

Reply:
Done 👍